### DISCUSSION TITLE
User names and passwords.

### DISCUSSION THREAD

**Discussion by MkStevo** [ravynsoft/ravynos#187] [General]
It was perhaps just me, and I'm not in any way a UNIX/Linux user so this may not be news to anyone else...

After I got airyxOS booting, when I installed it to the virtual hard drive, if I entered a user name with uppercase letters, I was unable to log in, every time I entered my user name and password I was asked (again, and again) for my user name and password. Only after I set BOTH my user name and password to all lowercase could I log in (which then happened automatically without me having to type either).

As I said, I'm new to this and I'm sure this is well known, but it fooled me for a while. The only way I could "fix" this was to try installing again, and again until I tried all lowercase. It might be coincidence it started working then, but worth a try?

**Comment by mszoek** [comment]
Ah, you probably hit bug #159 which inserts extra spaces when Shift or other modifier keys are used in the installer. Sorry about that.

**Reply by MkStevo** [reply]
Not a problem, it fooled me for ages, and I was ready to give up. I thought if I made a post about it, it would help me out in the future even if every one else remembers!
